README.md

```diff
@@ -10,52 +10,94 @@
 
 ```yml
-# dbt_project.yml
+vars:
+    quickbooks_database: your_destination_name
+    quickbooks_schema: your_schema_name 
+```
 
-...
-config-version: 2
+## Step 4: Enabling/Disabling Models
+<details><summary>Expand for details</summary>
+Your QuickBooks connector might not sync every table that this package expects. This package takes into consideration that not every QuickBooks account utilizes the same transactional tables.
 
+By default, most variables' values are assumed to be `true` (with exception of  `using_purchase_order` and `using_credit_card_payment_txn`). In other to enable or disable the relevant functionality in the package, you will need to add the relevant variables:
+
+```yml
 vars:
-    quickbooks_database: your_database_name
-    quickbooks_schema: your_schema_name
+  quickbooks_source:
+    using_address: false # disable if you don't have addresses in QuickBooks
+    using_bill: false # disable if you don't have bills or bill payments in QuickBooks
+    using_credit_memo: false # disable if you don't have credit memos in QuickBooks
+    using_department: false # disable if you don't have departments in QuickBooks
+    using_deposit: false # disable if you don't have deposits in QuickBooks
+    using_estimate: false # disable if you don't have estimates in QuickBooks
+    using_invoice: false # disable if you don't have estimates in QuickBooks
+    using_invoice_bundle: false # disable if you don't have estimates in QuickBooks
+    using_journal_entry: false # disable if you don't have estimates in QuickBooks
+    using_payment: false # disable if you don't have estimates in QuickBooks
+    using_refund_receipt: false # disable if you don't have estimates in QuickBooks
+    using_transfer: false # disable if you don't have estimates in QuickBooks
+    using_vendor_credit: false # disable if you don't have estimates in QuickBooks
+    using_sales_receipt: false # disable if you don't have estimates in QuickBooks
+    using_purchase_order: true # enable if you want to include purchase orders in your staging models
+    using_credit_card_payment_txn: true # enable if you want to include credit card payment transactions in your staging models
 ```
+</details>
+
+## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
+
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
+</details>
+
```

Comment:
This should be the step after the optional configs. So it should be step 6 in this README

Reply:
Updated!